### ISSUE TITLE
Accept regexp

### ISSUE THREAD

**Issue by beatrizdemiguelperez** [jkleinsc/broccoli-serviceworker#20]
It would be a nice if the addon accepts a regExp in the includePaths parameter of config/environment:
 with output toolbox.router.any(/google.com/, toolbox.cacheFirst);

**Comment by jkleinsc** [comment]
@beatrizdemiguelperez I agree.  I think it can added for the next release

**Comment by jkleinsc** [comment]
@beatrizdemiguelperez this is now available in the new version (0.1.0).  Regular Expressions can now be used for the new cacheFirstURLs, cacheOnlyURLs, fastestURLs and networkFirstURLs options.  For your use case, you would specify the following:

``` javascript
cacheFirstURLs: [
  /google.com/
]
OR
cacheFirstURLS: [{
  route: /google.com/,
  options: {
    origin: 'https://www.google.com/'
  }
}]
```
